src/utilities/identities/identities.ts

```diff
@@ -308,14 +303,16 @@ export async function importIdentity(
   const seed = mnemonicToMiniSecret(backupPhrase);
 
   const lightDidDocument = getLightDidFromSeed(seed);
-  const lightDid = lightDidDocument.uri;
+  const lightDid = lightDidDocument.id;
+
+  const {
+    didDocumentMetadata: { canonicalId, deactivated },
+  } = await DidResolver.resolve(lightDid, {});
```

Comment:
Isn’t the second parameter optional? I feel that it has to be.

Reply:
I also think it's weird that it's not optional, especially when it's called `ResolutionOptions`

Reply:
Let’s apply slight pressure to someone

Reply:
@rflechtner we are applying slight pressure to you 🙂

Reply:
That is an interesting one! I agree this is massively annoying and I'm inclined to make it optional - especially bc it is only required on the typing, but the actual implementation handles omitting it just fine. A quick look at the typings explains what feels like an oversight (Antonio built this, so I went to a similar discovery process):

```
  resolve: (
    /**
     * This is the DID to resolve.
     * This input is REQUIRED and the value MUST be a conformant DID as defined in 3.1 DID Syntax.
     */
    did: Did,
    /**
     * A metadata structure containing properties defined in 7.1.1 DID Resolution Options.
     * This input is REQUIRED, but the structure MAY be empty.
     */
    resolutionOptions: ResolutionOptions
  ) => Promise<ResolutionResult>
```
Meaning: it is a specification thing, which is why I am unsure what to do about it. I don't see any harm in making it optional really, but for some reason it's explicitly required even if empty...

Reply:
Okay, my opinion about this specification authors has sunk a little more, nothing to see here

Reply:
I think we don't need to be sticklers about it really, I'll put it in my list of improvements

Reply:
Not an oversight, but compliance to the resolution spec. We could make it optional, I agree.